source/administration/console/subnet-registration.rst

```diff
@@ -0,0 +1,89 @@
+.. _minio-console-subscription:
+
+===============================
+SUBNET Registration and Support
+===============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The MinIO Console provides access to subscription related tasks.
+Use the Console to subscribe to |SUBNET|, manage the deployment's SUBNET license, or access Support tools for sharing with MinIO Engineering.
+
+License
+-------
+
+MinIO offers two licensing options: 
+
+#. Open source with the :minio-git:`GNU AGPLv3 license <mc/blob/master/LICENSE>`
+#. Paid commercial license with included support direct from MinIO Engineers
+
+This page shows the current license status of the deployment.
+You can also begin the registration process to sign up for a paid subscription or add the deployment to an existing subscription.
+
+Deployments licensed under AGPLv3 must comply to the terms of the license.
+
+MinIO is Open Source
+~~~~~~~~~~~~~~~~~~~~
+
+We have designed MinIO as an Open Source software for the Open Source software community. 
+This requires applications to consider whether their usage of MinIO is in compliance with the GNU AGPLv3 license.
+
+MinIO cannot make the determination as to whether your application's usage of MinIO is in compliance with the AGPLv3 license requirements. 
+You should instead rely on your own legal counsel or licensing specialists to audit and ensure your application is in compliance with the licenses of MinIO and all other open-source projects with which your application integrates or interacts.
+
+We understand that AGPLv3 licensing is complex and nuanced. 
+It is for that reason we strongly encourage using experts in licensing to make any such determinations around compliance instead of relying on apocryphal or anecdotal advice.
+MinIO Commercial Licensing is the best option for applications which trigger AGPLv3 obligations (e.g. open sourcing your application). 
+Applications using MinIO - or any other OSS-licensed code - without validating their usage do so at their own risk.
```

Comment:
I did suggest using this, but reading it here feels heavy.

Suggest doing the following:

- Drop 29 through 33
- Move 35 through 36 up to 27
- Drop 38, 39
- Move 40 and 41 to be a graf after 27/28

Reply:
Oh, good, I also thought it heavy.